.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,17 @@
+name: Nightly
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+jobs:
+  test-gke:
+    name: Request GKE test cluster
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Request GKE test cluster
```

Comment:
Should we name this to reflect `NightlyPolicyStrees`, just so its easy to visualize the step in the Github Action UI?